### PR TITLE
composer update fail since not supporting anymore 2.4

### PR DESCRIPTION
**Pull request by jjborie**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Could you please update your PR and commit to have something like this one ? https://github.com/sonata-project/SonataCoreBundle/pull/240

File: composer.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
-        "sonata-project/admin-bundle": "~2.4",
-        "sonata-project/block-bundle": "~2.3",
+        "sonata-project/admin-bundle": "3.x-dev",
+        "sonata-project/block-bundle": "3.x-dev",
         "sonata-project/seo-bundle": "~2.0",
         "sonata-project/doctrine-extensions" : "~1.0",
         "sonata-project/cache-bundle": "^2.1.7",
         "sonata-project/cache": "^1.0.2",
-        "sonata-project/core-bundle": "^2.3.1",
+        "sonata-project/core-bundle": "3.x-dev",
```
Comment (Soullivaneuh) [inline composer.json]: Core bundle has a stable release now.

File: composer.json [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
         "symfony-cmf/routing-bundle": "~1.1",
-        "sonata-project/notification-bundle": "~2.2",
+        "sonata-project/notification-bundle": "~2.2|dev-patch-1",
```
Comment (Soullivaneuh) [inline composer.json]: Why ?

File: composer.json [hunk 3]
```diff
@@ -18,15 +18,15 @@
     ],
     "require": {
         "symfony/symfony": "~2.3|~3.0",
-        "sonata-project/admin-bundle": "~2.4",
-        "sonata-project/block-bundle": "~2.3",
+        "sonata-project/admin-bundle": "3.x-dev",
```
Comment (Soullivaneuh) [inline composer.json]: This bundle has a stable release now.

File: composer.json [hunk 4]
```diff
@@ -18,15 +18,15 @@
...
+        "sonata-project/block-bundle": "3.x-dev",
```
Comment (Soullivaneuh) [inline composer.json]: This bundle has a stable release now.

**Comment by alexkvak** [comment]
Made a new version #636

**Pull request closed without merging**